config/puma.rb

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env puma
+
+port Integer(ENV.fetch('PORT'))
```

Comment:
To make it fail early if no port was specified? E.g. if someone forgets to run under `foreman`.